examples/Esp8266-mqtt/ciotc_config.h

```diff
@@ -0,0 +1,72 @@
+/******************************************************************************
+ * Copyright 2018 Google
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+// This file contains your configuration used to connect to Cloud IoT Core
+
+// Wifi newtork details.
+const char *ssid = "SSID";
+const char *password = "PASSWORD";
+
+// Cloud iot details.
+const char *project_id = "project-id";
+const char *location = "us-central1";
+const char *registry_id = "my-registry";
+const char *device_id = "my-esp32-device";
+
+// To get the private key run (where private-key.pem is the ec private key
+// used to create the certificate uploaded to google cloud iot):
+// openssl ec -in <private-key.pem> -noout -text
+// and copy priv: part.
+// The key length should be exactly the same as the key length bellow (32 pairs
+// of hex digits). If it's bigger and it starts with "00:" delete the "00:". If
+// it's smaller add "00:" to the start. If it's too big or too small something
+// is probably wrong with your key.
+const char *private_key_str =
+"6e:b8:17:35:c7:fc:6b:d7:a9:cb:cb:49:7f:a0:67:"

```

Comment:
nit. prepend four-spaces as this is a continuation from the previous line

Reply:
done.